README.md

```diff
@@ -188,12 +188,12 @@ This project provides instructions for using Infracost in a CircleCI pipeline, i
           - run:
               name: Generate Infracost cost estimate baseline
               command: |
-                  infracost breakdown --path=${TF_ROOT} \
+                  infracost breakdown --path=/tmp/base \
```

Comment:
```suggestion
                  infracost breakdown --path=/tmp/base/${TF_ROOT} \
```

We do the same thing in https://gitlab.com/infracost/infracost-gitlab-ci, that ensures we run infracost in the correct dir at the checked-out git repo